tests/common.go

```diff
@@ -249,6 +249,11 @@ func StartVolDriverAndWait(appNodes []node.Node) {
 		Step(fmt.Sprintf("wait for volume driver to start on nodes: %v", appNodes), func() {
 			for _, n := range appNodes {
 				err := Inst().V.WaitDriverUpOnNode(n, Inst().DriverStartTimeout)
+				if err != nil {
```

Comment:
This is the candidate to have a function.

Reply:
Removed unnecessary debug message, no need to make a func() out of it